packages/medusa/src/api/routes/admin/products/update-variant.ts

```diff
@@ -134,7 +134,9 @@ export default async (req, res) => {
     ...validatedQueryParams,
   })
 
-  const [product] = await pricingService.setProductPrices([rawProduct])
+  const [product] = await pricingService.setProductPrices([rawProduct], {
```

Comment:
suggestion: remove the flag usage, the cache is invalidated anyway. We can keep the flag option as you mentioned but no need to use it here as it might collide with what the user is doing in the hook.